src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -57,15 +58,24 @@ class BasePaymentsPage extends React.Component {
         this.hidePasswordPrompt = this.hidePasswordPrompt.bind(this);
         this.navigateToTransferBalancePage = this.navigateToTransferBalancePage.bind(this);
         this.setMenuPosition = this.setMenuPosition.bind(this);
+        this.fetchData = this.fetchData.bind(this);
```

Comment:
I don't think we need to bind this here?

Reply:
Am 100% sure we dont, but all the methods used are bound so i followed suite. Just so I don't write something very different

Reply:
So I guess we can leave it like so as all the `bind` in the component are irrelevant.

Reply:
It's necessary in a few situations. Sometimes it's not. I think this is a situation where it's not needed.